tutorials/vpn-interop-guides/interop-guide-template/index.md

```diff
@@ -0,0 +1,1502 @@
+# Google Cloud VPN Interop Guide
```

Comment:
The template needs the standard metadata header for authors to fill in.

Reply:
Sorry, where can I find the standard metadata header?